ci-operator/step-registry/cri-o/node/e2e/nodeconformance/test/cri-o-node-e2e-nodeconformance-test-commands.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+set -xeuo pipefail
+
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/env"
+chmod +x "${SHARED_DIR}/login_script.sh"
+"${SHARED_DIR}/login_script.sh"
+
+timeout --kill-after 10m 400m ssh "${SSHOPTS[@]}" "${IP}" -- bash - <<EOF
+    SOURCE_DIR="/usr/go/src/github.com/cri-o/cri-o"
+    cd "\${SOURCE_DIR}/contrib/test/ci"
+    cat <<'PLAYBOOK' > e2e-nodeconformance.yml
+---
+- hosts: localhost, all
+  become: yes
+  environment:
+    GOPATH: /usr/go
+    KUBECONFIG: /var/run/kubernetes/admin.kubeconfig
+    CONTAINER_RUNTIME_ENDPOINT: "unix:///var/run/crio/crio.sock"
+  vars_files:
+    - "{{ playbook_dir }}/vars.yml"
+  tags:
+    - e2e-node
+  tasks:  
+    - name: build and install cri-o
+      include_tasks: "build/cri-o.yml"
+
+    - name: enable and start CRI-O
+      become: yes
+      systemd:
+        name: crio
+        state: started
+        enabled: yes
+        daemon_reload: yes
+    
+    - name: Disable selinux during e2e tests
+      command: 'setenforce 0'
```

Comment:
also an issue for this, we should fix node e2e to work with selinux

Reply:
Fixed it